css/properties/shape-outside.json

```diff
@@ -54,36 +50,100 @@
               "version_added": "24"
             },
             "safari": {
-              "prefix": "-webkit-",
-              "version_added": "8"
+              "version_added": "10.1"
             },
             "safari_ios": {
-              "prefix": "-webkit-",
-              "version_added": "8"
+              "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4"
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
         },
+        "image": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Shapes/Shapes_From_Images",
```

Comment:
As much as I love this guide, I wonder if maybe this ought to link to the [`<image>`](https://developer.mozilla.org/docs/Web/CSS/image) reference? Maybe @Elchi3 has an opinion on the intended use for the mdn_url on subfeatures?

In any case, the `en-US/` bit can be removed to get a pass from the linter

Reply:
+1 on the `<image>` URL. Thank you!